### ISSUE TITLE
Cache project and permissions data on startup

### ISSUE THREAD

**Issue by benjamingeer** [dhlab-basel/Knora#468]
Currently I believe project and permissions data are loaded from the triplestore and cached as they are requested. This seems to make the first login slower than subsequent logins. Would it be possible to load this data and cache it when Knora starts, the way we do with ontology data?

cc @tobiasschweizer

**Comment by subotic** [comment]
We probably could. This would require creating a UserProfileV1 for each registered user and storing it into the cache when `webapi` is started. But I don't think that this will alleviate the problem with the first login beeing slower. When a user is authenticated, then the same thing is executed for the first and any subsequent user.

But I will investigate this further (after the release).

**Comment by benjamingeer** [comment]
I think it probably doesn't make sense to do this for users (because there will be a lot of them), but I was thinking it could make sense to do it for project data and permission data (which there isn't so much of).

**Comment by subotic** [comment]
Yes, I agree. Caching projects and permissions is on my to-do list, but I wanted to finish implementing these responders first.
